template.rb

```diff
@@ -40,31 +32,42 @@ def apply_template!
   git :init unless preexisting_git_repo?
   empty_directory ".git/safe"
 
-  run_with_clean_bundler_env "bundle update"
-  run_with_clean_bundler_env "bin/rails webpacker:install"
-  install_dart_sass unless sprockets?
-  create_database_and_initial_migration
-  run_with_clean_bundler_env "bin/setup"
-
-  binstubs = %w[brakeman bundler bundler-audit rubocop sidekiq]
-  run_with_clean_bundler_env "bundle binstubs #{binstubs.join(' ')} --force"
-
-  template "rubocop.yml.tt", ".rubocop.yml"
-  run_rubocop_autocorrections
-
-  template "eslintrc.js", ".eslintrc.js"
-  template "prettierrc.js", ".prettierrc.js"
-  template "stylelintrc.js", ".stylelintrc.js"
-  add_yarn_lint_and_run_fix
-  add_yarn_start_script
-
-  unless any_local_git_commits?
-    git checkout: "-b main"
-    git add: "-A ."
-    git commit: "-n -m 'Set up project'"
-    if git_repo_specified?
-      git remote: "add origin #{git_repo_url.shellescape}"
-      git push: "-u origin --all"
+  after_bundle do
```

Comment:
This ensures that all of the commands in this block will run _after_ the default `rails new` setup has completed. That means importmaps-rails, webpacker, cssbundling-rails, etc. additions to the Gemfile, package.json, etc. will already be done. So we can safely build on top of what was created and not worry about conflicts or our files getting clobbered by those installers.